v2.2/recommended-production-settings.md

```diff
@@ -39,54 +39,89 @@ Term | Definition
-    - Use [zone configs](configure-replication-zones.html) to increase the replication factor from 3 (the default) to 5. This is especially recommended if you are using local disks rather than a cloud providers' network-attached disks that are often replicated underneath the covers, because local disks have a greater risk of failure. You can do this for the [entire cluster](configure-replication-zones.html#edit-the-default-replication-zone) or for specific [databases](configure-replication-zones.html#create-a-replication-zone-for-a-database), [tables](configure-replication-zones.html#create-a-replication-zone-for-a-table), or [rows](configure-replication-zones.html#create-a-replication-zone-for-a-table-or-secondary-index-partition) (enterprise-only).
-        {{site.data.alerts.callout_danger}}
-        {% include {{page.version.version}}/known-limitations/system-range-replication.md %}
-        {{site.data.alerts.end}}
+For best performance:
+
+- Use NVMe SSDs over HDDs.
+- To simulate a bare metal deployment where machines are directly connected to one physical disk each, create and mount a local SSD using a SCSI interface.
+- Use more powerful nodes. Adding more CPU is usually more beneficial than adding more RAM.
+- To calculate IOPS, use [sysbench](https://github.com/akopytov/sysbench). If IOPS decreases, add more nodes to your cluster to add IOPS.
+
+#### Resilience recommendations
+
+For more resilient clusters:
+
+- Use many smaller nodes instead of fewer larger ones. Recovery from a failed node is faster when data is spread across more nodes. We recommend using 8 cores (i.e., 16 threads), one per node.
+- Use [zone configs](configure-replication-zones.html) to increase the replication factor from 3 (the default) to 5 (across at least 5 nodes). This is especially recommended if you are using local disks with no RAID protection rather than a cloud providers' network-attached disks that are often replicated underneath the covers, because local disks have a greater risk of failure. You can do this for the [entire cluster](configure-replication-zones.html#edit-the-default-replication-zone) or for specific [databases](configure-replication-zones.html#create-a-replication-zone-for-a-database), [tables](configure-replication-zones.html#create-a-replication-zone-for-a-table), or [rows](configure-replication-zones.html#create-a-replication-zone-for-a-table-or-secondary-index-partition) (enterprise-only).
+    {{site.data.alerts.callout_danger}}
+    {% include {{page.version.version}}/known-limitations/system-range-replication.md %}
+    {{site.data.alerts.end}}
 
 ### Cloud-specific recommendations
 
 Cockroach Labs recommends the following cloud-specific configurations based on our own internal testing. Before using configurations not recommended here, be sure to test them exhaustively.
 
+In general, we have found CockroachDB to perform better on AWS over GCE.
```

Comment:
We need to be careful about wording--we don't want the takeaway to be dont use GCP.

Reply:
Also It should be GCP not GCE